src/main/java/panav/task/ToDo.java

```diff
@@ -0,0 +1,26 @@
+package panav.task;
+
+import panav.exception.ToDoDescriptionException;
+
+/**
+ * Class to represent an ToDo task. It contains attributes for it's description.
+ */
+public class ToDo extends Task {
+
+    /**
+     * Constructor to initialise attributes.
+     * @param description
+     * @throws ToDoDescriptionException
+     */
+    public ToDo(String description) throws ToDoDescriptionException {
+        super(description);
+        if (description.length() == 0) {
```

Comment:
Perhaps !description.length() is better?

Reply:
True, shall make the change